package.json

```diff
@@ -37,8 +37,15 @@
     ".": {
       "types": "./dist/src/index.d.ts",
       "import": "./dist/src/index.js"
+    },
+    "./alloc-unsafe.js": {
+      "browser": "./dist/src/alloc-unsafe-browser.js",
+      "default": "./dist/src/alloc-unsafe.js"
```

Comment:
Why `"default"` and not `"import"`?

Reply:
I agree `import` would make more sense. I am still getting familiar with the exports map. Thanks.